docs/basisterminologie.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: page-with-side-nav
+title: Basisterminologie klantinteracties
+date: 21-08-2023
+---
+
+# Basisterminologie klantinteracties
+
+Deze pagina beschrijft een toelichting bij kernconcepten die binnen het domein van klantinteracties worden erkend. Deze toelichting dient als inleiding tot en aanvulling bij het [semantisch informatiemodel Klantinteracties](informatiemodel/semantisch_model.md).
+
+# Begrippen
+In deze toelichting wordt een aantal begrippen genoemd. Het grootste deel daarvan is gedefinieerd in het informatiemodel Klantinteracties:
+- Klantcontact: een contact tussen een klant of een vertegenwoordiger van een klant en de gemeente;
+- Partij: een persoon of organisatie waarmee de gemeente een relatie heeft;
```

Comment:
Is deze definitie niet te breed ? We hebben het toch over personen of organisaties die contact hebben gehad met de gemeente ? En ook nog aangegeven hebben akkord te zijn met vastleggen als partij in het kader van hergebruik ?

Reply:
Het toestemming geven voor hergebruik is als uitkomst van de discussie van vorige week géén voorwaarde voor registratie van een partij. Ik zal dit verduidelijken.

Spreken van "het bestaan een relatie" betekent dat de mogelijkheid wordt opengehouden partijgegevens proactief, dus op het moment dat de gemeente weet dat zij met een persoon of organisatie contact gaat hebben, maar voordat dit contact daadwerkelijk plaatsvindt, te registeren. Ik kan me voorstellen dat het zinvol is deze mogelijkheid open te houden.